src/apps/products/services/product_services.py

```diff
@@ -47,18 +47,18 @@ def create_product(
             raise ServiceException("Wrong categories!")
 
         product_data["categories"] = categories
-
+    
     if product_data.get("inventory"):
         inventory_data = product_data.pop("inventory")
-        new_inventory = ProductInventory(quantity=inventory_data["quantity"])
-        session.add(new_inventory)
-        session.commit()
-        product_data["inventory_id"] = new_inventory.id
-
+    
     new_product = Product(**product_data)
+        
     session.add(new_product)
```

Comment:
solve there a case when json empty with 'inventory' key is being sent. If this will not be popped error may appear